### PR TITLE
Use React Router to Add A Header Component That Routes To Home, Playlists, and Quiz

### LINKED ISSUE #8: Use React Router to Add A Header Component That Routes To Home, Playlists, and Quiz.
## Description
- [x] Add a Header component.
- [x] Use React Router to route to `/` (home), `/playlists`, and `/quiz` inside the component.
- [x] Manually test that each route works.
- [x] At `/`, We should see something like "Welcome To Glow" on the home page.
- [x] At `playlists`, we should see all of the playlists.
- [x] At `quiz`, we should something like "Coming Soon".

## Resources
If you've not worked in React Router before, that's fine! This project is here to give you the opportunity to learn new things.  
1. https://medium.com/@pshrmn/a-simple-react-router-v4-tutorial-7f23ff27adf

### PR DESCRIPTION
**Pull request by karuna24s**
# Fixes
This PR fixes #8

# Before
There was no way for a user to route to home, playlists, and quiz.

# After
The user can now route to home, playlists, and quiz.

### CODE REVIEW

**Comment by LNA** [comment]
Thanks for submitting! I've set aside 20 minutes tomorrow to pair after our Zoom call.  

For your PR, let's update the format to something like:

# Fixes
the ticket that it fixes

# Before
The behavior before

# After
The behavior after

**Comment by karuna24s** [comment]
# Fixes
This PR fixes #8

# Before
The Glow app did not have a React router for a header component to route the Home , Playlists, and Quiz routes. Also note that header, home, and quiz components need to be created and manually test each route.

# After
Setup a React Router for a header component to route the Home, Playlists, and Quiz routes. Created header, home, and quiz components. Manually tested each route.

File: src/App.js [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
   render() {
     return (
       <div className="App">
         <div className="App-header">
-          <h1>Playlists</h1>
+          <Header />
         </div>
-        <PlaylistsContainer />
+        <Main />
+
```
Comment (LNA) [inline src/App.js]: This extra space can be removed 👍

**Comment by LNA** [comment]
Verified that the changes play well with the API.  LGTM!!! Merge when ready.

**Pull request merged**